packages/common/src/configuration/__snapshots__/service-configuration.spec.ts.snap

```diff
@@ -166,7 +166,7 @@ Object {
       "format": "int",
     },
     "minimumWaitTimeforScanResultQueryInSeconds": Object {
```

Comment:
can u also remove unknown from scan status. let scan property be null, if we don;t have the info

Reply:
how, we should we put if there's no info?

Reply:
ig there is no info set it to null instead of scanstatus unknown

Reply:
but this could be in a different pr